armi/physics/neutronics/isotopicDepletion/crossSectionTable.py

```diff
@@ -39,6 +39,10 @@ class CrossSectionTable(collections.OrderedDict):
     XStable is indexed by nucNames
     (nG), (nF), (n2n), (nA), (nP) and (n3n) are expected
     the cross sections are returned in barns
+
+    .. impl:: Generate cross section table.
+        :id: I_ARMI_DEPL_TABLES
+        :implements: R_ARMI_DEPL_TABLES
```

Comment:
Is this where this should be or is it on one of these? Or both? All three? 🤷 
1. https://github.com/terrapower/armi/blob/f2909b5588682ffc3592ad3d7ac6802e59611611/armi/physics/neutronics/isotopicDepletion/crossSectionTable.py#L160
2. https://github.com/terrapower/armi/blob/f2909b5588682ffc3592ad3d7ac6802e59611611/armi/physics/neutronics/isotopicDepletion/isotopicDepletionInterface.py#L120

Reply:
I'm not sure how this is typically used in practice so I'm not sure if it should go in just one and not the others, so I added it to all three to cover our bases.

Reply:
Sure, I like Tony's idea. 

I like both of those.

It probably means moving the test crumbs though?

Thanks!